src/easylink/step.py

```diff
@@ -1216,13 +1216,13 @@ class ParallelStep(TemplatedStep):
 
     @property
     def config_key(self):
-        """The pipeline specification key required for a ``ParallelStep``."""
-        return "parallel"
+        """The pipeline specification key required for a ``CloneableStep``."""
+        return "clones"
 
     @property
     def node_prefix(self):
-        """The prefix to be used in the ``ParallelStep`` node name."""
-        return "parallel_split"
+        """The prefix to be used in the ``CloneableStep`` node name."""
+        return "clone"
```

Comment:
And this is what changes the node name from using "parallel_split_{#}` to `clone_{#}` (which shows up in the results directory structure, DAG.svg (at the top of this PR, e.g.), etc.